### PR TITLE
Controller spec

### PR DESCRIPTION
**Pull request by anyepeter**
In this PR, I implemented the controllers testing with the following requirements 🔖 ;
### General requirements
✅ Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
✅ Make sure that you used correct [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).
✅ Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
### Ruby requirements
✅ Follow our list of [best practices for Ruby](https://github.com/microverseinc/curriculum-ruby/blob/main/articles/ruby_best_practices.md).
### Project requirements
- Add RSpec as a gem to your project.
✅ Create a Request spec file for all your controllers.
    - Make sure that for each action you check:
       ✅ If response status was correct.
       ✅ If a correct template was rendered.
       ✅ If the response body includes correct placeholder text.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -53,6 +53,7 @@ gem 'bootsnap', require: false
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
   gem 'debug', platforms: %i[mri mingw x64_mingw]
+  gem 'rails-controller-testing'
 end
```
Comment (FrederickMih) [inline Gemfile]: - [ ] Hi @anyepeter `rspec-rails` is the testing framework hence it is always advisable to place the gem in the `:development`, `:test` group in your Gemfile

**Review by FrederickMih — CHANGES_REQUESTED** [review]
## Hi :raising_hand:  @anyepeter, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!





## Highlights :green_circle: 

- PR is professional :+1: 
- Descriptive README.md file :+1: 
- Linter checks passed :heavy_check_mark: 


## Required Changes ♻️

- [ ] I usually run the rspec command on the root directory of the app and some of your previous tests are failing. Please kindly fix the minor issues.
- Screenshot:
Screenshot from 2023-03-07 11-03-06


_Check the comments under the review._

## Optional suggestions

- Hi @anyepeter, when you ran the generator `rails g rspec:install`, the Rspec generator generate app/spec directory where your rspec tests should resite. To keep our codebase clean, the directory app/test was supposed to be deleted.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by anyepeter** [comment]
I have adjusted the issues and everything is working now as expected 
Screenshot 2023-03-07 114208

**Review by addod19 — APPROVED** [review]
### Hi @anyepeter, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Highlights
- [ ] Nice job fixing the test :rocket: 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**